src/components/anonymousUserForm/AnonymousUserForm.js

```diff
@@ -0,0 +1,16 @@
+import styles from "@/styles/logInModal.module.scss";
+
+const AnonymousUserForm = ({setToLogin}) => {
+  return (
+    <>
+    <h2>Boka utan att logga in</h2>
+    <form>
+        <input required placeholder="Emailadress" type="email"></input>
+        <button className={styles.continueButton} type="submit">GÅ VIDARE</button>
```

Comment:
When you want uppercase text, I would recommend doing it so by text-transform: uppercase in the CSS rather than typing it out like this in the HTML. It makes it a lot easier to revert in the future if needed.

Reply:
I absolutely agree, a bit lazy of me, will fix.